ui/components/app/wallet-overview/coin-buttons.tsx

```diff
@@ -352,13 +375,29 @@ const CoinButtons = ({
       }
       ///: END:ONLY_INCLUDE_IF
       default: {
+        trackEvent(
```

Comment:
I think this is wrong. We already have this event above. Probably a bad conflict resolution? 😅

We are probably sending this event twice now.